serveradmin/servershell/static/js/servershell/autocomplete/command.js

```diff
@@ -117,6 +117,16 @@ $(document).ready(function() {
                         })
                     });
                 }
+                if (command === 'bookmark') {
+                    for (const key in localStorage) {
+                        if (localStorage.hasOwnProperty(key)) {
```

Comment:
Is this if necessary? Shouldn't `const key in localStorage` already give you just properties?

Reply:
Unfortunately yes. Otherwise you get also the object properties such as "length" etc.

Reply:
I see .. you might want to try this:
``` js
Object.keys(localStorage)
```
Found it here: https://stackoverflow.com/a/17748203/2625488

Reply:
Will do

Reply:
Features have been added. There is now bookmark, bookmarkadd and bookmarkdel command.

Reply:
Adjust to use the above method.